department-of-veterans-affairs/abd-vro
# Topic name prefixes in non dev environment are incorrect

## What was the problem?
Topic name prefixes in non dev environment are incorrect, need to be corrected based on [our wiki page](https://github.com/department-of-veterans-affairs/abd-vro/wiki/BIE-Kafka-Client#kafka-topic-prefixes)


## How does this fix it?[^1]
update non dev yaml files

## How to test this PR
- deploy to any non dev environment, and check `vro-xample-workflows` log to see if kafka events go through


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
